back/app/services/invalid_data_checker.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+class InvalidDataChecker
+  SKIP_CLASSES = %w[
+    EmailCampaigns::Delivery
+    CommonPassword
+    Analytics::FactVisit
```

Comment:
Just wondering if we should skip all the Analytics classes? They are all just views on other data, so if there is invalid data there it is going to be caused by other classes I would think

Reply:
Yes, ideally we shouldn't skip them. However, there was a bug one or two months ago where the script would fail to run validations for this specific class, that's why it was added to the classes to skip.